phoenix-core/src/it/java/org/apache/phoenix/end2end/ParameterizedIndexUpgradeToolIT.java

```diff
@@ -298,20 +299,22 @@ private void checkOldIndexingCoprocessors(String [] indexList, String [] tableLi
         }
     }
 
-    @Parameters(name ="IndexUpgradeToolIT_mutable={0},upgrade={1},isNamespaceEnabled={2}")
+    @Parameters(name ="IndexUpgradeToolIT_mutable={0},upgrade={1},isNamespaceEnabled={2}, rebuild={3}")
     public static synchronized Collection<Object[]> data() {
         return Arrays.asList(new Object[][] {
-            {false, false, true},
-            {true, false, false},
-            {false, true, false},
-            {true, true, true}
+            {false, false, true, false},
```

Comment:
Don't you want to test false, false, true, true  and
true, false, false, false?

Reply:
rebuild parameter is related to the upgrade parameter. So I covered all the 4 permutations of upgrade and rebuild.